src/components/app/data/services/enterpriseCustomerUser.test.js

```diff
@@ -309,3 +310,22 @@ describe('fetchInProgressPathways', () => {
     expect(response.status).toEqual(200);
   });
 });
+describe('postUnlinkUserFromEnterprise', () => {
+  const mockEnterpriseCustomerUserUUID = 'test-enterprise-customer-user-uuid';
+  const mockUserEmail = 'test-user@example.com';
+  const UNLINK_USER_ENDPOINT = `${APP_CONFIG.LMS_BASE_URL}/enterprise/api/v1/enterprise-customer/${mockEnterpriseCustomerUserUUID}/unlink_users/`;
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    axiosMock.reset();
+  });
+
+  it('passes correct POST body', async () => {
+    axiosMock.onPost(UNLINK_USER_ENDPOINT).reply(200, {});
```

Comment:
[nit/curious] Is there a response body for this API endpoint on success? If not, should we leave off the `{}`, similar to `axiosMock.onPost(SAVE_CSOD_LEARNER_PARAMS_ENDPOINT).reply(200);`?